tests/operators/test_batch.py

```diff
@@ -186,16 +186,15 @@ def test_data_consistency(
 
     res = from_interface(res, array_interface)
     res2 = from_interface(res2, array_interface)
-    slope_err = 1e-3
-    r_value_err = 1e-3
+    atol = 1e-3
+    rtol = 1e-3
     # FIXME 2D Sense is not very accurate...
     if len(operator.shape) == 2 and operator.uses_sense:
         print("Reduced accuracy for 2D Sense")
-        slope_err = 1e-1
-        r_value_err = 1e-1
+        atol = 1e-1
+        atol = 1e-1
 
-    for i in range(len(res)):
-        assert_correlate(res[i], res2[i], slope_err=slope_err, r_value_err=r_value_err)
+    npt.assert_allclose(res, res2, atol=atol, rtol=rtol)
```

Comment:
Moved assert_correlate to assert_allclose. @paquiteau to just confirm if we could be missing any tests due to assert_correlate which can be changed in this PR itself.